libs/SalesforceSDKCore/SalesforceSDKCore/SFSDKWindowManagerTests.m

```diff
@@ -0,0 +1,65 @@
+//
+//  SFSDKWindowManagerTests.m
```

Comment:
Ideally, we could abstract the windowing interfaces to drive more unit tests around the logic for window ordering, priorities, etc.